### ISSUE TITLE
The action 'Move Messages' is enabled for one SAS attached queue

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#5892]
**Storage Explorer Version**: 1.25.0-dev
**Build Number**: 20220630.8
**Branch**: main
**Platform/OS**: Windows 10/Linux Ubuntu 20.04/MacOS Monterey 12.4 (Apple M1 Pro)
**How Found**: From running test case
**Regression From**: Previous release (1.24.3)

## Steps to Reproduce ##
1. Expand one storage account -> Queues.
2. Create a queue -> Add one message.
3. SAS attach the queue -> Observe the toolbar of the attached queue.
4. Check whether the action 'Move Messages' is disabled.

## Expected Experience ##
The action 'Move Messages' is disabled.
![image](https://user-images.githubusercontent.com/41351993/176622209-7dc1bc91-73e1-40ac-b12b-6722c8ecb478.png)

## Actual Experience ##
The action 'Move Messages' is enabled.
![image](https://user-images.githubusercontent.com/41351993/176622226-4061710f-553b-4e60-bd3c-278d022be3ef.png)

## Additional Context ##
Here is the screenshot if clicking 'Move Messages'.
![image](https://user-images.githubusercontent.com/41351993/176622455-952143c9-d748-4b07-b91e-fa8084ec2b14.png)

**Comment by MRayermannMSFT** [comment]
Issue is that the default action for the dropdown, move all messages, is always enabled.
Thoughts:
- Fix the move all messages can execute
- Maybe chance default to move selected as well?

**Comment by craxal** [comment]
@MRayermannMSFT Actually, the "Move Selected" command is the default action, which is always enabled. We just need to add the same enable logic to that command as the "Move All" command.

**Comment by v-xianya** [comment]
@craxal 
Verified this issue on the build main/20220706.1.
The action 'Move Messages' is disabled. The little arrow is enabled. Is this intentional?
![image](https://user-images.githubusercontent.com/41351993/177450280-93e7ff99-c560-4f8e-857c-03ae9f800991.png)

**Comment by craxal** [comment]
@v-xianya Yes, this intentional. The dropdown part should always be enabled, even if the menu options it shows are all disabled.
